packages/drivers/local-driver/src/localDocumentService.ts

```diff
@@ -96,6 +98,8 @@ export class LocalDocumentService implements IDocumentService {
 			ordererToken.jwt,
 			client,
 			this.localDeltaConnectionServer.webSocketServer,
+			undefined,
+			this.logger,
```

Comment:
So this is a behavior change where we'll start collecting more logs whereas before they were going to the ether?  Nice to separate that in a change this big.